spec/factories/grant.rb

```diff
@@ -116,14 +117,14 @@
 
     factory :new_grant,                                     traits: %i[new]
     factory :new_grant_with_users,                          traits: %i[new with_users]
-    factory :draft_grant,                                   traits: %i[draft with_users_and_submission_form with_reviewer]
+    factory :draft_grant,                                   traits: %i[draft with_users_and_submission_form with_reviewer with_panel]
     factory :published_grant,                               traits: %i[published]
     factory :open_grant_with_users,                         traits: %i[published open with_users]
     factory :closed_grant_with_users,                       traits: %i[closed with_users]
     factory :published_open_grant,                          traits: %i[published open]
     factory :published_open_grant_with_users,               traits: %i[published open with_users_and_submission_form with_panel]
     factory :published_closed_grant,                        traits: %i[published closed]
-    factory :published_closed_grant_with_users,             traits: %i[published closed with_users]
+    factory :published_closed_grant_with_users,             traits: %i[published closed with_users_and_submission_form with_submission with_reviewer with_panel]
```

Comment:
these factory names seem a little arbitrary and they don't always indicate the traits they contain. For example, :grant_with_users and :draft_grant_with_users have panel, but neither does :new_grant_with_users nor ::open_grant_with_users. As I suggested in chat, instead of creating a new factory for each different combination,  how about add traits when actually constructing an instance in specs? See my comments in spec/system/panels/submissions/submission_spec.rb

Reply:
yeah, the names are now convoluted. i'm a bit reluctant to go the route you suggest because a panel is now created on grant creation. 

i think the whole file needs to be reworked given current state of the app rather than the add-as-we-go model we've been using.

Reply:
will be dealing w/ factory clean up in #644